### ISSUE TITLE
encrypted PGP/MIME compatibility

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-android#354]

**Comment by tomholub** [comment]
![image](https://user-images.githubusercontent.com/6306961/47673123-a8a3d580-dbab-11e8-996d-3b175403c9ac.png)

**Comment by tomholub** [comment]
Basically, as you can see, the content of decrypted message can itself be a MIME message. That's why it's called PGP/MIME.

The rough steps are as follows:
```Java
PgpDecrypted decrypted = js.crypto_message_decrypt(data);
String decryptedString = decrypted.getString();
if (js.resembles_mime_message(decryptedString.substr(0, 1000)) { // just 1kb is enough to determine
  MimeMessage decoded = js.mime_decode(decryptedString);
  if (decoded.getHtml() != null) {
    render_content_as_html(decoded.getHtml());
  } else if(decoded.getText() != null) {
    render_content_as_text(decoded.getText());
  } else { // else show the mime string
    render_content_as_text(decryptedString);
  }
  Attachment[] renderableInnerAttachments = [];
  Attachment[] renderableInnerPubkeys = [];
  for (att of decoded.getAttachments()) {
    if (att.treat_as() != 'public_key') {
      renderableInnerAttachments.push(att);
    } else {
      renderableInnerPubkeys.push(att.as_text());
    }
  }
  if (renderableInnerAttachments.length) {
    render_inner_attachments(renderableInnerAttachments);
  }
  if (renderableInnerPubkeys.length) {
    render_public_keys(renderableInnerPubkeys)
  }
} else {
   // not a MIME message process as before
}

```


Todo for me:
 - surface `js.resembles_mime_message`
 - explain in more detail

You can see how these messages are displayed by FlowCrypt Extension, just check `encrypted/html` folder and go through a few messages.

**Comment by tomholub** [comment]
I'm moving this to future milestone so that we can get 0.6.9 out soon

**Comment by tomholub** [comment]
moving this to 0.7.1 because it will be easier to do with #358 done

**Comment by tomholub** [comment]
fixed in 0.8.1
